### PR TITLE
Add a function to cleanly convert Params to Labels.

### LINKED ISSUE #1790: Feature: MakeStudyInfo()
## Feature Details
Add a function that takes the same input as `Report_StudyInfo()`, and returns a df with `Description` + `Value`, like the one produced as `study_status_table` in `Report_StudyInfo()`

## Additional Comments
I want to use this for the `Site Metadata` module in {gsm.app}. In general I think we'll probably want to split up functions for prepping the data, building a pretty table, and printing that table in a report.

### PR DESCRIPTION
**Pull request by jonthegeek**
## Overview
Function to add a `Labels` column to `dfGroups`.

Closes #1790.

## Test Notes/Sample Code

Notes:

### CODE REVIEW

**Comment by jonthegeek** [comment]
@lauramaxwell This one is ready to review, and will be useful in {gsm.app} in lots of places! (I don't need to export another function, I don't think, despite what I just said)

File: R/util-MakeParamLabels.R [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#'   parameters.
+#'
+#' @examples
+#' head(gsm::reportingGroups)
+#' MakeParamLabels(head(gsm::reportingGroups))
+#' MakeParamLabels(
+#'   head(gsm::reportingGroups),
+#'   list(ParticipantCount = "Number of Participants")
+#' )
+#' MakeParamLabelsList(head(gsm::reportingGroups$Params))
```
Comment (lauramaxwell) [inline R/util-MakeParamLabels.R:22]: tiny typo in this example
```suggestion
#' MakeParamLabelsList(head(gsm::reportingGroups$Param))
```

**Review by lauramaxwell — CHANGES_REQUESTED** [review]
Looks great! The tiniest typo in the example in the header and then we're gtg!

**Review by lauramaxwell — APPROVED** [review]
👍

**Pull request merged**